misc/ballerinalang-data-mapper/src/test/java/org/ballerinalang/datamapper/util/TestUtil.java

```diff
@@ -64,17 +81,150 @@
  */
 public class TestUtil {
 
+    private static final String HOVER = "textDocument/hover";
+
+    private static final String CODELENS = "textDocument/codeLens";
+
+    private static final String COMPLETION = "textDocument/completion";
+
+    private static final String SIGNATURE_HELP = "textDocument/signatureHelp";
+
+    private static final String DEFINITION = "textDocument/definition";
+
+    private static final String REFERENCES = "textDocument/references";
+
+    private static final String RENAME = "textDocument/rename";
+
+    private static final String EXECUTE_COMMAND = "workspace/executeCommand";
+
     private static final String CODE_ACTION = "textDocument/codeAction";
 
+    private static final String FORMATTING = "textDocument/formatting";
+
+    private static final String IMPLEMENTATION = "textDocument/implementation";
+
+    private static final String DOCUMENT_SYMBOL = "textDocument/documentSymbol";
+
+    private static final String WORKSPACE_SYMBOL_COMMAND = "workspace/symbol";
+
     private static final String WORKSPACE_CONFIG_CHANGE = "workspace/didChangeConfiguration";
 
     private static final Gson GSON = new Gson();
 
-    private static final WorkspaceManager workspaceManager = new BallerinaWorkspaceManager();
-
     private TestUtil() {
     }
 
+    /**
+     * Get the textDocument/hover response.
+     *
+     * @param filePath        Path of the Bal file
+     * @param position        Cursor Position
+     * @param serviceEndpoint Service Endpoint to Language Server
+     * @return {@link String}   Response as String
+     */
+    public static String getHoverResponse(String filePath, Position position, Endpoint serviceEndpoint) {
```

Comment:
Shall we remove unnecessary methods from Test Util? Please revisit all methods in this class

Reply:
Removed unnecessary methods